resolwe_bio/tools/cuffnorm_group_labels.py

```diff
@@ -1,16 +1,15 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
 # pylint: disable=missing-docstring,invalid-name
 # XXX: Refactor to a comand line tool and remove pylint disable
 """Create group labels from replicates."""
-from __future__ import absolute_import, division, print_function
 
 import argparse
 
 parser = argparse.ArgumentParser(description="Create group labels")
 parser.add_argument('replicates', help="Define replicates and groups")
 args = parser.parse_args()
 
-replicates = set(args.replicates.split(","))
```

Comment:
Is it ok to sort `replicates` here and not [here](https://github.com/genialis/resolwe-bio/pull/499/commits/5ea34fa6c678bc1070faa165f566847913eea52e#diff-e00ba752209ec3fee45885177c8237f9R15)?

Reply:
Yes. Here they must be sorted because of the 'set', otherwise the group outputs might differ every time the process is ran. In the second file it is correct not to sort them since they must have the same order as other inputs (samples, files) to the process.

Reply:
Yes. Here they must be sorted because of the 'set', otherwise the group outputs might differ every time the process is ran. In the second file it is correct not to sort them since they must have the same order as other inputs (samples, files) to the process.